File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.java
class Solution {
    public String solution(int n) {
        String su = "수";
        String bak = "박";
        
        StringBuilder stringBuilder = new StringBuilder();
        for (int i = 0; i < n; i++) {
            if (i % 2 == 0) {
                stringBuilder.append(su);
            } else {
                stringBuilder.append(bak);
            }
        }
        
        return stringBuilder.toString();
    }
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.kt
class Solution {
    fun solution(n: Int): String {
        val su = "수"
        val bak = "박"
        
        val stringBuilder = StringBuilder()
        for ( i in 0 until n){
            if (i % 2 == 0){
                stringBuilder.append(su)
            } else {
                stringBuilder.append(bak)
            }     
        }
        
        return stringBuilder.toString()
    }
}